frontend/catalyst/jit.py

```diff
@@ -239,6 +239,47 @@ def circuit(x: int):
         be unrolled during tracing, "copy-pasting" the body 5 times into the program rather than
         appearing as is.
 
+    .. details::
+        :title: In-place JAX array updates with Autograph
```

Comment:
Not sure why but this section is not being rendered at all: https://xanaduai-pennylane--1143.com.readthedocs.build/projects/catalyst/en/1143/code/api/catalyst.qjit.html

The only difference I can see right now is the other `details` section's `title` has an empty line below. Maybe try that?

Reply:
Hopefully it is fixed now 🤞